packages/fxa-settings/src/lib/gql.test.ts

```diff
@@ -80,4 +83,26 @@ describe('errorHandler', () => {
 
     expect(window.location.replace).not.toHaveBeenCalled();
   });
+
+  it('does not redirect if the current page does not require the user to be authenticated', () => {
```

Comment:
I think this is going to cause problems if we write tests after this that depend on window location. See what I had to do in `CookiesDisabled` tests to cleanup after, I think you just need a `beforeAll` and `afterAll` in this `it`.